cypress/e2e/ondemand/sid/metrics.cy.js

```diff
@@ -0,0 +1,43 @@
+import { NAVIGATION, loadHomepage, navigateToMetrics } from "../../../support/utils/navigation.js";
+import {changeProfile} from "../../../support/utils/profiles";
+
+describe('FASRC Dashboard - Metrics Widget and Page', () => {
+  const fasrcClusterProfile = Cypress.env('fasrc_cluster_profile')
+  Cypress.config('baseUrl', NAVIGATION.baseUrl);
+
+  before(() => {
+    loadHomepage()    
+    changeProfile(cy.sid.profiles.sid.title)
+  })
+
+  beforeEach(() => {
+    //DEFAULT SIZE FOR THESE TESTS
+    cy.viewport(cy.sid.screen.largeWidth, cy.sid.screen.height)
+    loadHomepage()
+  })
+
+  describe(`${fasrcClusterProfile}: Should display Metrics page`, () => {
```

Comment:
For Sid, we only need the name of the test, no need to display `${fasrcClusterProfile}: `
For consistency, you can take a look at other Sid test to compare